webserver/static/scripts/datasets/eval-jobs-viewer.js

```diff
@@ -89,13 +105,34 @@ var EvaluationJobsViewer = React.createClass({
         });
         window.location.hash = "";
     },
+    handleJobDelete: function (jobID, index) {
+        $.ajax({
+            type: "DELETE",
+            url: "/datasets/" + container.dataset.datasetId + "/" + jobID,
+            success: function (data, textStatus, jqXHR) {
+                alert("Evaluation job " + jobID + " has been removed from the queue.")
```

Comment:
it'll go away from the list if it succeeds. If it fails, perhaps you could add a message inline?